integrations/varnish/documentation.yaml

```diff
@@ -0,0 +1,143 @@
+exporter_type: sidecar
+app_name_short: Varnish
+app_name: {{app_name_short}}
+app_site_name: Varnish
+app_site_url: https://varnish-cache.org/
+exporter_name: the Varnish exporter
+exporter_pkg_name: prometheus_varnish_exporter
+exporter_repo_url: https://github.com/jonnenauha/prometheus_varnish_exporter
+dashboard_available: true
+minimum_exporter_version: "1.6.1"
+multiple_dashboards: false
+dashboard_display_name: {{app_name_short}} Prometheus Overview
+config_mods: |
+  apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: varnish
+    namespace: default
```

Comment:
Let's omit namespace here... not super critical but just for consistency with all the other yamls where we don't specify it and ask the user to run `kubectl apply -f <FILE> -n <NAMESPACE>`